src/components/UserIcon/UserIcon.styles.js

```diff
@@ -0,0 +1,38 @@
+import { css } from 'styled-components'
+import styled from 'styled-components/macro'
+
+import { hoverState, mediaQueryTabletLandscapeOnly } from '../../library/styling/mediaQueries'
+import { IconUser } from '../icons'
+import theme from '../../theme'
+
+export const UserProfilePicture = styled.img`
+  width: ${theme.typography.largeIconSize};
+  height: ${theme.typography.largeIconSize};
```

Comment:
this is mostly just copy paste, but I changed the icon size to be all the same so they work in a table regardless of what user icon type shows (fallback, picture, or initials)